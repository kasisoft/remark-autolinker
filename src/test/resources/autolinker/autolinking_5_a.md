---
title: Make sure longer terms won't be "shadowed" by shorter ones
---
Spring Boot is an awesome framework based on Spring and everything can be done in Java.
