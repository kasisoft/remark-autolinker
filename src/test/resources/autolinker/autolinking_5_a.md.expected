---
title: Make sure longer terms won't be "shadowed" by shorter ones
---

[Spring Boot](https://spring.io/projects/spring-boot) is an awesome framework based on [Spring](https://spring.io/) and everything can be done in [Java](https://www.java.com/de/).
